spec/carrierwave/storage/aws_file_spec.rb

```diff
@@ -44,6 +44,22 @@
     end
   end
 
+  describe '#read' do
+    let(:s3_object) { double('Aws::S3::Object') }
+    before          { aws_file.file = s3_object }
+
+    it 'reads the retrieved body if called without block' do
+      expect(s3_object).to receive_message_chain('get.body.read')
+      aws_file.read
+    end
+
+    it 'does not retrieve body if block given' do
+      proc = Proc.new { }
```

Comment:
`proc` is a method on Kernel and equivalent to `Proc.new`, a different name would be better.